features/netinfo.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-10-21
-name: Network Information API
-description: TODO
+name: Network Information
+description: The `NetworkInformation` interface provides information about the connection a device is using to communicate with the network and provides a means for scripts to be notified if the connection type changes.
```

For consistency elsewhere, I think we can name the primary entrypoint instead of the interface. Plus, some conciseness suggestions.

```suggestion
description: The `navigator.connection` API provides information about the network connection a device is using and fires events when the connection type changes.
```